gymnasium/dev_wrappers/rescale_action.py

```diff
@@ -0,0 +1,80 @@
+"""Wrapper for rescaling actions to within a max and min action."""
+from typing import Union
+
+import jumpy as jp
+import numpy as np
```

Comment:
I'm not sure we need to bother now, but I think that we should look to only use jumpy where we add the necessary functions

Reply:
It is used in https://github.com/Farama-Foundation/Gymnasium/blob/54b99ac8ac2194937e58e45efddf5d06922069e1/gymnasium/dev_wrappers/rescale_action.py#L80 maybe I am missing something?

Reply:
Good point, I was more looking at the code for np.less_equal in the constructor so I think this might be fine